FreeRTOS/FreeRTOS
# Removes old empty Demo directories

Description
-----------
There were few empty directories and this pull request cleans them.

Test Steps
-----------
None

Related Issue
-----------
Clutter


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
